src/app/GitExtUtils/GitUI/Theming/ColorHelper.cs

```diff
@@ -21,7 +21,7 @@ public static Color DimColor(Color color)
             const uint maskWithoutLeastSignificantBits = 0xFE_FE_FE_FE;
             uint defaultBackground = (uint)SystemColors.Window.ToArgb();
             int dimCode = (int)((((uint)color.ToArgb() & maskWithoutLeastSignificantBits) >> 1) + ((defaultBackground & maskWithoutLeastSignificantBits) >> 1));
-            return Color.FromArgb((dimCode >> 16) & 0xff, (dimCode >> 8) & 0xff, dimCode & 0xff);
+            return Color.FromArgb((dimCode >> 24) & 0xff, (dimCode >> 16) & 0xff, (dimCode >> 8) & 0xff, dimCode & 0xff);
```

Comment:
The alpha channel should rather not be "dimmed". Although the effect is negligible: `(255, 255) => 254` and `(254, 254) => 254`.

Reply:
If there is a alpha, it should be mixed, but rounding error causes 255->254
But the alpha in the original can be reused as SystemColors.Window has alpa 255.